### DISCUSSION TITLE
Can it use OracleDB and Customize the data value when add subject.

### DISCUSSION THREAD

**Discussion by PoomCPS** [exadel-inc/CompreFace#944] [Q&A]
1. Can it use OracleDB 
2. Is it possible customize the data value when add subject like,
ID, SunjectName >> ID, SubjectFname, SubjectLname
or Can I custom the API of the project

**Comment by pospielov** [comment]
1. No, we use some PostgreSQL features, so there is no simple way to use OracleDB
2. No, I explained the reasoning here: https://github.com/exadel-inc/CompreFace/issues/934
Of course, you can add this by yourself in the code and build CompreFace from your code.

**Reply by PoomCPS** [reply]
That mean I need to add my custom field and api to the java code and UI then build my custom docker-image and change the docker-image in docker compose file. or I can build the whole project docker-image by one file

**Reply by pospielov** [reply]
I would recommend staying with a docker-compose file.
We still have a build with one dockerfile: https://github.com/exadel-inc/CompreFace/blob/master/custom-builds/Single-Docker-File/Dockerfile
More considerations on what version to choose you can read here: https://github.com/exadel-inc/CompreFace/blob/master/docs/Installation-options.md

**Reply by PoomCPS** [reply]
@pospielov  can I ask more question? 
if need to create another container I work with javascriptSDK that will link to use the service from compreface 
how I set the url to make it can call the compreface service, I already add 2 of them to the same container but look like it can't connect

or it have another better way?
